google/cloud/bigquery/opentelemetry_tracing.py

```diff
@@ -0,0 +1,115 @@
+# Copyright 2020 Google LLC All rights reserved.
```

Comment:
No "All rights reserved" in recommended header https://opensource.google/docs/releasing/preparing/#Apache-header

```suggestion
# Copyright 2020 Google LLC
```